velox/exec/tests/TableScanTest.cpp

```diff
@@ -215,6 +215,34 @@ TEST_F(TableScanTest, allColumns) {
   EXPECT_LT(0, exec::TableScan::ioWaitNanos());
 }
 
+TEST_F(TableScanTest, connectorStats) {
+  auto hiveConnector =
+      std::dynamic_pointer_cast<connector::hive::HiveConnector>(
+          connector::getConnector(kHiveConnectorId));
+  EXPECT_NE(hiveConnector, nullptr);
+  auto cacheStats = hiveConnector->fileHandleCacheStats();
+  EXPECT_EQ(cacheStats.curSize, 0);
```

Comment:
nit: ASSERT_XX put expected value after the actual value but EXPECT_XX put the expected value before the actual value.